wagtailvideos/wagtail_hooks.py

```diff
@@ -53,7 +53,12 @@ def editor_js():
     return format_html(
         """
         <script>
+        if (window.chooserUrls) {{
```

Comment:
are the double curlys for a reason? It's valid js but not something I've seen before

Reply:
I just realised it's in a `format_html`  call and not just straight js